ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePage.tsx

```diff
@@ -76,19 +76,16 @@ export const imageCveSummaryQuery = gql`
 export const imageCveAffectedImagesQuery = gql`
     ${imagesForCveFragment}
     # by default, query must include the CVE id
-    query getImagesForCVE(
-        $query: String
-        $imageListPagination: Pagination
-        $imageComponentPagination: Pagination
-    ) {
+    query getImagesForCVE($query: String, $pagination: Pagination) {
         imageCount(query: $query)
-        images(query: $query, pagination: $imageListPagination) {
+        images(query: $query, pagination: $pagination) {
```

Comment:
We aren't doing pagination of the nested table queries, so I'm changing this naming to the standard "pagination" parameter.